projects/Gibbed.MassEffect2.FileFormats/SaveFile.cs

```diff
@@ -20,126 +20,15 @@
  *    distribution.
  */
 
+using Gibbed.IO;

```

Comment:
Cleaned up the usings weirdly? Needs sorting + removal of empty lines.

Reply:
done